Brainfuck/spark/program.yaml

```diff
@@ -0,0 +1,11 @@
+source: src/spark.adb
+
+prepare:
+  - alr build 
+  - alr gnatprove
+
+run: ./bin/spark {PROGRAM}
+
+authors:
+  - eatonphil
```

Comment:
No need to have me in here, it's understood that the different languages are ports. Just your name is fine! :)

Reply:
🙏